Reactions/ReactorBase.cpp

```diff
@@ -1,8 +1,140 @@
 #include "ReactorBase.H"
 
+#include <arkode/arkode_arkstep.h>
```

Comment:
Should these go in `ReactorBase.H` @marchdf ?

Reply:
this led me into a rabbit hole of organizing the sundials headers which is not easy.

Reply:
Ok no big deal

Reply:
yeah it's a pain... I tried to have the headers be included in the files they are actually used. But it's hard...